res/database/Default/user/jorgaeff.xml

```diff
@@ -1773,8 +1773,8 @@
 				<en>Macrohard introduces "Proof 1.01"</en>
 			</title>
 			<description>
-				<de>Die von Makrohart endgültig freigegebene Version Proof 1.01 wurde auf einer Pressekonferenz im südschwedischen Helsingborg vorgestellt. Das Betriebssystem benötigt einen Arbeitsspeicher von 256 KByte und ein 5.25"-Diskettenlaufwerk.</de>
-				<en>The version Proof 1.01 finally released by Macrohard was presented at a press conference in Helsingborg, southern Sweden. The operating system requires a memory of 256 KByte and a 5.25" floppy drive.</en>
+				<de>Die von Makrohart endgültig freigegebene Version Proof 1.01 wurde auf einer Pressekonferenz im südschwedischen Helsingborg vorgestellt. Das Betriebssystem benötigt einen Arbeitsspeicher von 256 KByte und ein 5.25″-Diskettenlaufwerk.</de>
```

Comment:
I think here the "regular" upper quotes are OK!?

Reply:
yeah, probably. I found the font in the game has problem with such special characters anyway.

Reply:
If some quot styles are missing:
- not all fonts have them (while we use a one which should)
- identify the chars and pass me the UTF8-code of it so I can debug more easily